docs/source/reference/numpysupported.rst

```diff
@@ -614,7 +614,23 @@ execution logic.
 
 The following :py:class:`Generator` methods are supported:
 
+* :func:`numpy.random.Generator().exponential()`
+* :func:`numpy.random.Generator().gamma()` (*)
+* :func:`numpy.random.Generator().normal()` (*)
 * :func:`numpy.random.Generator().random()`
+* :func:`numpy.random.Generator().standard_normal()`
+* :func:`numpy.random.Generator().standard_exponential()`
+* :func:`numpy.random.Generator().standard_gamma()` (*)
+* :func:`numpy.random.Generator().uniform()` (*)
+
+.. note::
+  Users can expect Numba to be bit accurate with Numpy within
+  the last few (at max 5) ULPs for Linux-x86_64, Windows and macOS.
+
+  For distributions marked above with `(*)`: Due to certain compiler
+  technicalities, there might be discrepanies of order of 1000 ULPs
+  on 32-bit architectures as well as linux-aarch64 and linux-ppc64le platforms.
+  For more details, `see the comment here <https://github.com/numba/numba/pull/8038#issuecomment-1165571368>`
```

Comment:
```suggestion
  For distributions marked above with `(*)`: Due to instruction selection
  differences across compilers, there may be discrepancies in the order of
  1000s of ULPs on 32-bit architectures as well as linux-aarch64 and
  linux-ppc64le platforms. The differences are unlikely to impact the "quality"
  of the random number generation as they occur through changes in
  rounding that happen when fused-multiply-add is used instead of multiplication
  followed by addition.
```
suggest noting the impact on the RNG quality and offering a brief explanation to users.